iceoryx_examples/iceperf/uds.hpp

```diff
@@ -42,19 +43,28 @@ class UDS : public IcePerfBase
     static constexpr int32_t INVALID_FD = -1;
 
     UDS(const std::string& publisherName, const std::string& subscriberName) noexcept;
+    /// @brief Cleans up outdated sockets, e.g. from a previous test
+    /// @attention only leader is allowed to call this
+    static void cleanupOutdatedResources(const std::string& publisherName, const std::string& subscriberName) noexcept;
+
     void initLeader() noexcept override;
     void initFollower() noexcept override;
     void shutdown() noexcept override;
 
   private:
+    static constexpr const char* PREFIX{"/tmp/"};
```

Comment:
This will not be working in windows. Is this alright for now?

Reply:
Well, the whole UDS test doesn't work at the moment. I think we can adjust this once we figured out how to do it on Windows. As a side note, this PR just moved the `PREFIX` from another place to the class itself and didn't introduce it.